src/videojs.markers.js

```diff
@@ -1,9 +1,25 @@
+/*! videojs-markers - v0.4.0 - 2015-03-13
+* Copyright (c) 2015 ; Licensed  */
 /*! videojs-markers !*/
 'use strict'; 
 
 (function($, videojs, undefined) {
    //default setting
-   var defaultSetting = {
+   var defaultSetting = { 
```

Comment:
The reason for this abstraction was to allow pre-existing array of objects with similar data but different attributes to be used without converting the array object's format. For example, I have an angular app that pulls in large transcription files from s3 and allows editing of those transcripts.  The transcript files contain arrays of objects representing words and includes text and date attributes, however the text and date attributes are named differently (and require data transformation in some transcription formats).  By adding this abstraction, I can directly pass the transcription word array directly into addMarkers without performing a translation of the transcript file beforehand.   This then allows multiple ui components to act the same data array, which is useful for two-way data binding.